lib/atomic_lti/open_id_middleware.rb

```diff
@@ -81,6 +81,7 @@ def call(env)
             if payload
               decoded_jwt = JWT.decode(id_token, nil, false)[0]
```

Comment:
It feels strange to decode without validation even though we know it's valid.  Isn't payload already the validated and decoded token?